bumble/device.py

```diff
@@ -1781,9 +1781,9 @@ def on_connection_encryption_key_refresh(self):
             pass
 
     @dataclass
-    class Parameters:
-        connection_interval: float  # Connection interval, in milliseconds. [LE only]
-        peripheral_latency: int  # Peripheral latency, in number of intervals. [LE only]
+    class Parameters:  # LE only
```

Comment:
I think we should use docstring.
```suggestion
    class Parameters:
      '''LE-only connection parameters.
      
      Args:
        connection_interval: ...
        peripheral_latency: ...
      '''
```